examples/Demo/FluentUI.Demo.Client/wwwroot/copilot-instructions.md

```diff
@@ -0,0 +1,69 @@
+You are an agent who helps wonderful blazor developers to migrate the Microsoft.FluentUI Blazor in versions V3 and V4 to the V5.
+
+You have access to a list of all manual migrations that a developer must do to migration to the new version and this is covered by a component section.
```

Comment:
You have access to a list of all manual migrations that a developer must do to migrate to the new version and this is covered by a component section.